Backend/dev/migrations/rider-app/1262-adding-ticket-booking-cancellation.sql

```diff
@@ -0,0 +1,35 @@
+
+WITH MerchantMessages AS (
+  SELECT T1.merchant_id, 'TICKET_BOOKING_CANCELLED', 'Sorry, {#personName#} your Ticket Service {#serviceId#} had been cancelled. booking amount will be refunded shortly, please check app for more information', T1.id
+  FROM atlas_app.merchant_operating_city AS T1
+)
+INSERT INTO atlas_app.merchant_message (merchant_id, message_key, message, merchant_operating_city_id)
+  (SELECT * FROM MerchantMessages);
+
+-- DONT RUN BELOW QUERIES IN PRODUCTION FOR NOW
+
+CREATE INDEX idx_ticket_booking_short_id ON atlas_app.ticket_booking USING btree (short_id);
+CREATE INDEX idx_ticket_booking_person_id ON atlas_app.ticket_booking USING btree (person_id);
+CREATE INDEX idx_ticket_booking_visit_date ON atlas_app.ticket_booking USING btree (visit_date);
+
+CREATE INDEX idx_ticket_booking_service_short_id ON atlas_app.ticket_booking_service USING btree (short_id);
+CREATE INDEX idx_ticket_booking_service_ticket_booking_id ON atlas_app.ticket_booking_service USING btree (ticket_booking_id);
+CREATE INDEX idx_ticket_booking_service_service_id ON atlas_app.ticket_booking_service USING btree (ticket_service_id);
+
+CREATE INDEX idx_ticket_booking_service_category_id ON atlas_app.ticket_booking_service_category USING btree (id);
+CREATE INDEX idx_ticket_booking_service_category_ticket_booking_service_id ON atlas_app.ticket_booking_service_category (ticket_booking_service_id);
+CREATE INDEX idx_ticket_booking_service_category_service_category_id ON atlas_app.ticket_booking_service_category (service_category_id);
+
+
+CREATE INDEX idx_ticket_booking_people_category_serviceCategoryId ON atlas_app.ticket_booking_people_category (ticket_booking_service_category_id);
+CREATE INDEX idx_ticket_booking_people_category_people_category_id ON atlas_app.ticket_booking_people_category (people_category_id);
+
+CREATE INDEX idx_ticket_service_placesId ON atlas_app.ticket_service (places_id);
+
+CREATE INDEX idx_special_occasion_entityId_date ON atlas_app.special_occasion (entity_id, date);
+
+CREATE INDEX idx_business_hour_categoryId ON atlas_app.business_hour (category_id);
+
+CREATE INDEX idx_seat_management_date_categoryId ON atlas_app.seat_management (date, ticket_service_category_id);
+
+CREATE INDEX idx_ticket_place_merchant_operating_city_id ON atlas_app.ticket_place (merchant_operating_city_id);
```

Comment:
This much indexing will make insertion very slow. Let's have it only for 1-2 columns per table

Reply:
okay sure